src/platform/user/profile/utilities/index.js

```diff
@@ -88,18 +88,22 @@ export function mapRawUserDataToState(json) {
       ? vet360ContactInformation
       : mockContactInformation,
     session,
+    veteranStatus: {
+      status: veteranStatus?.status,
+      isVeteran: veteranStatus?.isVeteran,
+      servedInMilitary: veteranStatus?.servedInMilitary,
+    },
   };
 
   if (meta && veteranStatus === null) {
     const errorStatus = meta.errors.find(
       error => error.externalService === commonServices.EMIS,
     ).status;
-    userState.veteranStatus = getErrorStatusDesc(errorStatus);
+    userState.veteranStatus.status = getErrorStatusDesc(errorStatus);
```

Comment:
This makes sense to me. I don't think we should be making the entire `veterarnStatus` object a simple string when there is an error.